src/common/docker-compose/index.ts

```diff
@@ -1,10 +1,31 @@
 import fs from 'fs-extra';
+import _ from 'lodash';
 import path from 'path';
-import DependencyManager, { DatastoreNode, ServiceNode } from '../../dependency-manager/src';
+import DependencyManager, { DatastoreNode, DependencyNode, ServiceNode } from '../../dependency-manager/src';
 import { ExternalNode } from '../../dependency-manager/src/graph/node/external';
 import { LocalServiceNode } from '../dependency-manager/local-service-node';
 import DockerComposeTemplate from './template';
 
+const inject_params = (environment: { [key: string]: string | number }, data_node: DependencyNode, ) => {
+  const param_prefix = `$ARC_${data_node.name.replace(/[^\w\s]/gi, '_').toUpperCase()}_${data_node.tag.replace(/[^\w\s]/gi, '_').toUpperCase()}`;
+  const host_param_placeholder = `${param_prefix}_HOST`;
```

Comment:
Does add a backslash to the $ not work as opposed to adding a module?
`\${param_prefix}_HOST`;

Reply:
That would escape the entire variable that we're trying to insert into the string, wouldn't it?

Reply:
What is the output of escapeRegExp?

Reply:
Fixed